packages/core/src/lib/implementation/runner.ts

```diff
@@ -26,7 +31,8 @@ export async function executeRunnerConfig(
   });
 
   // read process output from file system and parse it
-  const outputs = await readJsonFile(path.join(process.cwd(), outputFile));
+  const outputs = await readJsonFile(outputFile);
+  await removeDirectoryIfExists(path.dirname(outputFile));
```

Comment:
This deserves a comment, took me a while to realize why you remove the directory after reading.

```suggestion
  // clean up plugin output directory
  await removeDirectoryIfExists(path.dirname(outputFile));
```